dashboard/app/public_json_api.go

```diff
@@ -45,20 +46,31 @@ func getExtAPIDescrForBugPage(bugPage *uiBugPage) *publicAPIBugDescription {
 			}
 			return []string{bugPage.Bug.ExternalLink}
 		}(),
-		FixCommits: func() []fixCommit {
+		FixCommits: func() []vcsCommit {
 			if len(bugPage.Bug.Commits) == 0 {
 				return nil
 			}
-			var res []fixCommit
+			var res []vcsCommit
 			for _, commit := range bugPage.Bug.Commits {
-				res = append(res, fixCommit{
+				res = append(res, vcsCommit{
 					Title: commit.Title,
 					Hash:  commit.Hash,
 					Link:  commit.Link,
 				})
 			}
 			return res
 		}(),
+		CauseCommit: func() *vcsCommit {
+			if bugPage.BisectCause == nil || bugPage.BisectCause.Commit == nil {
+				return nil
+			}
+			commit := bugPage.BisectCause.Commit
+			return &vcsCommit{
+				Title: commit.Title,
+				Hash:  commit.Hash,
```

Comment:
Don't we need some description of the git tree with the hash? Hash on its own it mostly meaningless. What can I do with it?

Reply:
Right.
Looking [here](https://syzkaller.appspot.com/bug?id=d7e411c5472dd5da33d8cc921ccadc747743a568&json=1), Title + Link should be enough. In our case Link == repo+hash.

Reply:
But it's not machine-usable and API is for machines.

Reply:
[Here](https://github.com/google/cvelist/blob/cve-automation/test/syzkaller/two-valid-cves/0-api.json), the hash was used, thus it is needed.
Got your point. I'll add the repo link to generalize API.